src/basic/helpers.js

```diff
@@ -0,0 +1,34 @@
+import Utils from '../utils/utils';
+
+var BaseHelpers = {};
+
+let BH = BaseHelpers;
+
+export default BaseHelpers;
+
+const OPACITY = {
+	line: 'stroke-opacity',
```

Comment:
ups... I'm changing from one project to other and I forgot to change the setting, we need a editorconfig.